python-sdk/src/astro/databases/base.py

```diff
@@ -246,6 +247,7 @@ def create_table_using_schema_autodetection(
         file: File | None = None,
         dataframe: pd.DataFrame | None = None,
         columns_names_capitalization: ColumnCapitalization = "original",  # skipcq
+        pandas_options: PandasLoadOptions | None = None,
```

Comment:
@pankajastro could we name this `load_options`, using the same argument as the Databricks implementation?